src/components/uploader/js/Uploader.js

```diff
@@ -206,7 +207,13 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         var numFilesInBatch = batch.files.length;
         var fileLabelStr = fluid.uploader.fileOrFiles(that, numFilesInBatch);
 
-        var uploadingSize = batch.totalBytesUploaded + that.queue.sizeOfUploadedFiles();
+        var uploadingSize = that.queue.sizeOfUploadedFiles();
+        if (previoussizeOfUploadedFiles !== uploadingSize){
```

Comment:
Can you elaborate the reason behind this solution of wiping off ```batch.totalBytesUploaded``` when ```previoussizeOfUploadedFiles !== uploadingSize```? In what circumstances (use cases) would this be hit?